docs/_config.yml

```diff
@@ -0,0 +1,24 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: ConnectorX
+author: SFU-DB
+logo: logo.png
+
+# Force re-execution of notebooks on each build.
+# See https://jupyterbook.org/content/execute.html
+execute:
+  execute_notebooks: force
+
+
+# Information about where the book exists on the web
+repository:
+  url: https://github.com/Wukkkinz-0725/connector-x.git  # Online location of your book
```

Comment:
should we update the url and branch here?

Reply:
Yes, it is my fault. I will check other part now.